components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -129,10 +129,11 @@ class PlacesApi(path: String) : PlacesManager, AutoCloseable {
         }
     }
 
-    override fun importVisitsFromFennec(path: String) {
+    override fun importVisitsFromFennec(path: String): JSONObject {
         rustCall(this) { error ->
-            LibPlacesFFI.INSTANCE.places_history_import_from_fennec(
+            val json = LibPlacesFFI.INSTANCE.places_history_import_from_fennec(
                 this.handle.get(), path, error)
+            JSONObject(json)
```

Comment:
Does this string need to be "freed" i.e. [getAndConsumeRustString](https://github.com/mozilla/application-services/blob/eccb024c7273e1126005872056c7601572f525e6/components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt#L575)?

Reply:
hm I wonder if we can use `rustCallForString` here...